:zap: sync_telegram: added the ability to send files (#265)
itpp-labs/sync-addons: sync_telegram/doc/index.rst

Usage
=====

> 

Reviewer: ![delete](https://user-images.githubusercontent.com/186131/120976698-e3624700-c772-11eb-8ab5-24b437304a10.jpg)

* RESULT: the message is delivered via bot

You can continue chatting this way

> 

Reviewer: ```suggestion
```

File sending and receiving
--------------------------

> The operator has the ability to send not only text messages, but also messages with attachments.

Reviewer: ```suggestion
The operator has the ability to send not just text messages, but also messages with attachments.
```

> The Telegram Bot API has a limit on the files sent through the bot:

Reviewer: Добавь ссылку на пруф по словом лимит

 - The operator cannot send to Telegram user more than **10 MB for photos** and **50 MB for other files**.
 - Also, the operator cannot receive **files larger than 20 MB** from a Telegram user.

Subscribing telegram user to chatter
------------------------------------

> After the bot has created a record, you can also send messages to your partner directly from the record page.
> To do this, you need to add a discussion channel with the partner to the partner record.
> Find a button on the record page: ``Show Followers -> Add Channels``.

Reviewer: ```suggestion
After the bot has created a record, you can also send messages to telegram user directly from the record page (Lead, Task, etc.).
In order to do that, subscribe corresponding channel to the record.
Use button ``Show Followers -> Add Channels``.
```
